_posts/2011-12-21-amin11a.md

```diff
@@ -2,14 +2,14 @@
 title: Bandits, Query Learning, and the Haystack Dimension
 abstract: Motivated by multi-armed bandits (MAB) problems with a very large or even
   infinite number of arms, we consider the problem of finding a maximum of an unknown
-  target function by querying the function at chosen inputs (or arms).We give an analysis
+  target function by querying the function at chosen inputs (or arms). We give an analysis
   of the query complexity of this problem, under the assumption that the payoff of
   each arm is given by a function belonging to a known, finite, but otherwise arbitrary
   function class. Our analysis centers on a new notion of function class complexity
-  that we callthe \emphhaystack dimension, which is used to prove the approximate
-  optimality of a simple greedy algorithm. This algorithm is then usedas a subroutine
-  in a \parametric MAB algorithm, yielding provably near-optimal regret. We provide
-  a generalization to the infinite cardinality setting, andcomment on how our analysis
+  that we call the haystack dimension, which is used to prove the approximate
+  optimality of a simple greedy algorithm. This algorithm is then used as a subroutine
+  in a functional MAB algorithm, yielding provably near-optimal regret. We provide
```

Comment:
Following the main tex
<img width="424" alt="Screenshot 2022-02-10 at 19 53 24" src="https://user-images.githubusercontent.com/7121753/153392461-ee277f1c-13e9-43da-bdf5-dcab084d14bb.png">

PDF: https://github.com/mlresearch/v19/blob/gh-pages/amin11a/amin11a.pdf